binstar_client/utils/projects/__init__.py

```diff
@@ -0,0 +1,102 @@
+import inspect
+import logging
+import os
+from os.path import basename, isdir, join, relpath
+from binstar_client.errors import BinstarError
+from .filters import filters
```

Comment:
if we extract it into a library, that's easy to fix, just do a project find and replace, no problem.
I don't like relative imports because if you decide to move one file, it breaks everything. It also creates ambiguity, as in, which `models` is this importing? you have to look at the code tree and that's bad. code should read on it's on, without having to leave the file to find data.